controller/cache/cache.go

```diff
@@ -670,25 +670,29 @@ func (c *liveStateCache) IsNamespaced(server *appv1.Cluster, gk schema.GroupKind
 func (c *liveStateCache) IterateHierarchyV2(server *appv1.Cluster, keys []kube.ResourceKey, action func(child appv1.ResourceNode, appName string) bool) error {
 	clusterInfo, err := c.getSyncedCluster(server)
 	if err != nil {
-		return err
+		return fmt.Errorf("failed to get synced cluster %s: %w", server.Name, err)
```

Comment:
`getSyncedCluster` already has a similar message when it returns the error 

<img width="795" height="212" alt="Image" src="https://github.com/user-attachments/assets/23e8172f-67c8-4d6f-8ae3-778127c450a2" />

so I'm not sure if we want to add this twice